docs/third-party-developers/extensibility/data-store/checkout.md

```diff
@@ -0,0 +1,116 @@
+# wc/store/checkout
+
+## Table of Contents
+
+-   [Selectors](#selectors)
+    -   [getCustomerId](#getcustomerid)
+    -   [getCheckoutState](#getcheckoutstate)
+    -   [getOrderNotes](#getordernotes)
+    -   [hasError](#haserror)
+    -   [hasOrder](#hasorder)
+    -   [isAfterProcessing](#isafterprocessing)
+    -   [isBeforeProcessing](#isbeforeprocessing)
+    -   [isCalculating](#iscalculating)
+    -   [isComplete](#iscomplete)
+    -   [isIdle](#isidle)
+    -   [isProcessing](#isprocessing)
+
+## Selectors
+
+### getCustomerId
+
+Returns the ID of the customer.
+
+#### _Returns_
+
+-   `number`: ID of the customer.
+
+### getCheckoutState
+
+Returns the checkout state.
+
+#### _Returns_
+
+-   _state_ `Object`: Checkout state.
+
+### getOrderNotes
+
+Returns the order notes.
+
+#### _Returns_
+
+-   `string`: Order notes.
+
+### hasError
+
+Returns true if an error occurred, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an error occurred.
+
+### hasOrder
+
+Returns true if an order exists, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order occurred.
+
+### isAfterProcessing
+
+Returns true if an order had just been processed, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order had just been processed.
+
+### isBeforeProcessing
+
+Returns true if an order is about to be processed, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order is about to be processed.
+
+### isCalculating
+
+Returns true if any of the totals, taxes, shipping, etc. need to be calculated, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether the order needs calculation.
+
+### isComplete
+
+Returns true if an order is complete, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order is complete.
+
+### isIdle
+
+Returns true if the checkout state has changed, but no activity happened, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether the checkout state has changed, but no activity happened.
+
+### isProcessing
+
+Returns true if an order is processing, and false otherwise.
+
+#### _Returns_
+
+-   `boolean`: Whether an order is processing.
```

Comment:
Perhaps "Whether the checkout is processing" is slightly better,  WC orders have a "processing" status don't they and this could potentially confuse people

Reply:
Thanks for your suggestion. I've updated the wording accordingly.